examples/python-cicd-with-cloudbuilder/cloudbuild.yaml

```diff
@@ -1,13 +1,22 @@
+# This cloudbuild.yaml file uses the custom cloud builder image called python-cloudbuild that can be built from
+# the python-cloud-builder/ directory to run python CI steps
+# In this example the file runs pytest and pylint.  User added steps can be incorporated as well.
+
```

Comment:
Same notes above about comments formatting in here.

Reply:
I'm not sure I'm handling multiline comments right in python.